pkg/storage/chunk/schema_config.go

```diff
@@ -128,6 +132,26 @@ func defaultRowShards(schema string) uint32 {
 	}
 }
 
+// ChunkPathPeriod is introduced as a SchemaConfig value in Schema "v12"
+func defaultChunkPathPeriod(schema string) time.Duration {
+	switch schema {
+	case v12:
+		return 1 * time.Hour
```

Comment:
Since it may take a while to scale up prefixes, I think we should increase the default 
```suggestion
		return 12 * time.Hour
```